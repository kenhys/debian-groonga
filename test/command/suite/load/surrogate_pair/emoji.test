table_create Characters TABLE_HASH_KEY ShortText
column_create Characters unicode COLUMN_SCALAR ShortText

load --table Characters
[
{"_key": "\uD83C\uDF7A", "unicode": "U+1F37A BEER MUG"}
]

select Characters --output_columns _key,unicode
