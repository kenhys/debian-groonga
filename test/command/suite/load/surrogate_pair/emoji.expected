table_create Characters TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Characters unicode COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Characters
[
{"_key": "\uD83C\uDF7A", "unicode": "U+1F37A BEER MUG"}
]
[[0,0.0,0.0],1]
select Characters --output_columns _key,unicode
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_key",
          "ShortText"
        ],
        [
          "unicode",
          "ShortText"
        ]
      ],
      [
        "üç∫",
        "U+1F37A BEER MUG"
      ]
    ]
  ]
]
